language/documentation/examples/diem-framework/move-packages/DPN/tests/ValidatorConfigTests.move

```diff
@@ -121,7 +121,7 @@ module DiemFramework::ValidatorConfigTests {
     }
 
     #[test(tc = @TreasuryCompliance, dr = @DiemRoot)]
```

Comment:
I think it would generally better we have an extensible syntax like `abort_code = "VC:5"`.

Reply:
You can do that too (see example in the summary) :) There just aren't any constants for all of these diem cases, and it didn't seem necessary to add them